pyrex.ini

```diff
@@ -103,6 +103,7 @@ envvars =
 #   --device /dev/kvm
 args =
 	--device /dev/kvm
+	--security-opt seccomp=unconfined
```

Comment:
Actually, I don't like enabling this option in the developer `pyrex.ini`. I think it makes better security sense to have host machines which encounter the same errors as @gratian's machine, add this option to their local INI and not check this into the mainline for everyone.

The `.../azdo/.../pyrex.ini` file I'm less worried about, since that should only be used on our machines.

Reply:
Ok, removed `pyrex.ini` changes.